psalm.xml

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0"?>
 <psalm
-    errorLevel="8"
+    errorLevel="7"
     resolveFromConfigFile="true"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
-    errorBaseline="psalm-baseline.xml"
 >
     <projectFiles>
-        <directory name="src/Mutator" />
```

Comment:
didn't realize why we scan src/Mutator directory only

Reply:
We didn't use Psalm for anything else but to guard against `@psalm-mutation-free`